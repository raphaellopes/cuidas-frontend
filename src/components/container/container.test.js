// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import Container from './index';

const component = renderer.create(
  <Container>
    My children text
  </Container>,
);

describe('<Container />', () => {
  test('Should render the component', () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
