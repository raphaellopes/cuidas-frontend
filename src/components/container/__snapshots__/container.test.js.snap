// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Should render the component 1`] = `
<div
  className="sc-bwzfXH ZoqEH"
>
  <header
    className="sc-htpNat dplynj"
  >
    <a
      href="/"
    >
      <img
        alt="Cuidas"
        className="sc-bdVaJa katCQl"
        src="logo.svg"
      />
    </a>
  </header>
  <div>
    My children text
  </div>
</div>
`;
